treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -3,7 +3,7 @@
   fields:
     framework: 1
     keywords: perf, regression, talos-regression
-    default_component: General
+    default_component: Performance
     default_product: Testing
     cc_list: dave.hunt@gmail.com, rwood@mozilla.com, igoldan@mozilla.com, fstrugariu@mozilla.com
```

Comment:
Changes look good.

Now that we're here, could we also add Alex Ionescu & Marian to the `cc_list` from all frameworks?
Alex's email: alexandru.ionescu@softvision.ro
Marian's email: marian.raiciof@softivision.ro

Reply:
Done

Reply:
Thanks!